construction_progress_management/data/ir_model_fields.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="field_sale_order_line_delivered_progress" model="ir.model.fields">
+        <field name="ttype">float</field>
+        <field name="field_description">Delivered Progress</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="name">x_delivered_progress</field>
+        <field name="depends">qty_delivered,product_uom_qty</field>
+        <field name="compute"><![CDATA[
+for sol in self: sol['x_delivered_progress'] = False if not sol.product_uom_qty else 100 * (sol.qty_delivered / sol.product_uom_qty)
```

Comment:
I find it more intuitive to define the interesting computation first, and the fallback afterwards
```suggestion
for sol in self: sol['x_delivered_progress'] = 100 * (sol.qty_delivered / sol.product_uom_qty) if sol.product_uom_qty else False
```